public/views/cart.jsx

```diff
@@ -15,53 +15,104 @@
 
 'use strict';
 
-var Layout = require('./layout.jsx');
 var React = require('react');
+var Router = require('react-router');
 
+var ps = require('pubsub-js');
+var store = require('../store');
+var $ = require('jquery');
 
 module.exports = React.createClass({
+	getInitialState: function () {
+		return store && store.cart || this.props.cart;
+	},
+	componentDidMount: function () {
+		var self = this;
+		this.cartUpdate = ps.subscribe('cartUpdate', function (msg, items) {
+			//self.handleCartUpdate(items);
```

Comment:
The cartWidget is a part of the navigation, and is visible on the other pages as well.